### PR TITLE
dolt-init: add page

### PR DESCRIPTION
**Pull request by Torfab**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Torfab** [comment]
PR for issue #5471

File: pages/common/dolt-init.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# dolt init
+
+> Create an empty Dolt data repository.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-config>.
+
+- Create an empty Dolt data repository in the current directory:
+
+`dolt init`
+
+- Specify details name on the initial commit:
```
Comment (kbdharun) [inline pages/common/dolt-init.md:10]:
```suggestion
- Specify details on the initial commit:
```

Comment (Torfab) [reply pages/common/dolt-init.md]: little slip on last update, fixed in de9e53eaef9d7566340cc11ea34aeface4764d09

File: pages/common/dolt-init.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# dolt init
+
+> Create an empty Dolt data repository.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-config>.
```
Comment (navarroaxel) [inline pages/common/dolt-init.md:4]: This link doesn't work for me
```suggestion
> More information: <https://docs.dolthub.com/cli-reference/cli#dolt-config>.
```

File: pages/common/dolt-init.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# dolt init
+
+> Create an empty Dolt data repository.
```
Comment (navarroaxel) [inline pages/common/dolt-init.md:3]: The 2nd example isn't an empty repo, right?
```suggestion
> Initialize a new Dolt data repository.
```

File: pages/common/dolt-init.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-config>.
+
+- Create an empty Dolt data repository in the current directory:
+
+`dolt init`
+
+- Specify details on the initial commit:
```
Comment (navarroaxel) [inline pages/common/dolt-init.md:10]: Is this creating the first commit?

File: pages/common/dolt-init.md (hunk 2 above)
Comment (kbdharun) [reply pages/common/dolt-init.md:4]: Shouldn't it be 
```suggestion
> More information: <https://docs.dolthub.com/cli-reference/cli#dolt-init>.
```

File: pages/common/dolt-init.md (hunk 3 above)
Comment (Torfab) [reply pages/common/dolt-init.md:3]: I took this sentence from the official documentation.

the init commit push metadata only.

Solved in 2b940581aa22cd07b8b4b2703dd5e3daabeb8b4a

File: pages/common/dolt-init.md (hunk 2 above)
Comment (Torfab) [reply pages/common/dolt-init.md]: you are totally right, I've got it wrong with my other branch, fixed in 2b940581aa22cd07b8b4b2703dd5e3daabeb8b4a

File: pages/common/dolt-init.md (hunk 4 above)
Comment (Torfab) [reply pages/common/dolt-init.md:10]: this create a first commit of pure metadata in a .dolt directory

Comment (navarroaxel) [reply pages/common/dolt-init.md:10]:
```suggestion
- Initialize a new Dolt data repository creating a commit with the specified metadata:
```

Comment (Torfab) [reply pages/common/dolt-init.md]: solved in 565108342595f9a73cdf474ac80034772f3aa5fb

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**